CHANGELOG.md

```diff
@@ -129,6 +129,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Add integration testing for duplicate granule workflows. This includes new
     specs and workflows in the `ingestGranule`, `discoverGranules`,
     `lzardsBackup`, `cnmWorkflow`, and `orca` specs.
+- **CUMULUS-4162**
+  - Made a `includeTimestampHashKey` config variable for the `add-unique-granuleId` and `parse-pdr` task to configure whether the hashKey is  `collectionId` or `collectionId` + timestamp
+  - Updated documentation to detail the two different `hashKey` configuration options
```

Comment:
Jonathan has a google document about generateUniqueGranuleId which uses different language, we need to update it and add to Cumulus Documentation.